pages/your_tasks/gitlab_zenodo.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Publishing software from GitLab to Zenodo
+search_exclude: true
+description: How to obtain a DOI via Zenodo for your code releases on GitLab
+contributors: ["Thomas Vuillaume"]
+page_id: gitlab_to_zenodo
+related_pages: [releasing_code, github_zenodo, codemeta]
+keywords: [zenodo, gitlab, code release]
+# More information on which page id you can use can be found at https://rdmkit.elixir-europe.org/website_overview
+---
+<!-- Please take in mind our style guide https://rdmkit.elixir-europe.org/style_guide when writing the content of this page. -->
+
+
+## How can I get a Digital Object Identifier (DOI) for my code release on GitLab?
+
+### Description 
+
+Publishing your code to Zenodo is a valuable practice that enhances the visibility, citability, and long-term preservation of your research software. By linking your code repository to Zenodo, you can:
+
+1. Obtain a Digital Object Identifier (DOI) for your software, making it easier to cite and track.
+2. Ensure long-term preservation of your code, even if the original repository becomes unavailable.
+3. Increase the discoverability of your software through Zenodo's search functionality and metadata.
+4. Comply with funding agency requirements for open science and data sharing.
+5. Provide a stable, versioned archive of your software for reproducibility purposes.
+
+In this document, we describe how to automatically archive a snapshot of your code repository on Zenodo whenever you create a new release on GitLab. 
+
+### Considerations
+
+* You use a GitLab instance (either on gitlab.com or self-hosted) as your software repository
+* You make software releases (e.g. using [semantic versioning][semantic-versioning])
+* You use or are happy to make an account on the [Zenodo][zenodo] archive
+* You would like to archive your code on Zenodo and obtain a DOI for each software release that people can use to cite your work
+* You can provide metadata for your software in a `.zenodo.json` file or a `codemeta.json` file in the root of your repository to specify the metadata for your Zenodo deposit.
+* Your GitLab repository can run CI/CD pipelines.
+
+### Solutions
+
+* Provide software metadata in a `.zenodo.json` or `codemeta.json` file.
+  * Create a `.zenodo.json` file in the root directory of your repository following the [Zenodo's metadata schema](https://developers.zenodo.org/#metadata).
```

Comment:
Why not link to the codemeta tutorial as an alternative to the zenodo.json? It seems inconsistent to point  only to the Zenodo one when in the previous sentence we recommend both files. 

I would prioritize having people add CodeMeta to their repos, since it has more support than a single platform

Reply:
Oh, I would not drop it, but clarify that for gitlab2zenodo you need a zenodo.json file. And how to conver to it through eossr tool.